### ISSUE TITLE
DPL-336 - As a developer I would like to improve the style of the landing page to make it easier on the eye and easier to use [C=S, V=3]

### ISSUE THREAD

**Issue by stevieing** [sanger/traction-ui#751]
**User story**
As a developer I would like to improve the style of the landing page to make it easier on the eye and easier to use. We have now added a header and footer so it would be good to pretty the boxes for each of the pipeline

**Who are the primary contacts for this story**
Steve / Seena / Sangeetha

**Acceptance criteria**
To be considered successful the solution must allow:
- [x] A wireframe to allow us to create a simple visual representation of the style to support the best approach
- [x] A style for each pipeline box with a suitable border
- [x] A style for each pipeline box description
- [x] A style for each hyperlink to each page e.g. reception sample etc.
- [x] A style for the text within each hyperlink
- [ ] An icon for each hyperlink (optional / nice to have)

**Dependencies**
This story is blocked by the following dependencies:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

References
This story has a non-blocking relationship with:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

**Additional context**

**Comment by stevieing** [comment]
@seenanair just a bit of feedback:

- would it be possible for the traction text to be white to give sufficient colour contrast
- would it be possible for the link text to be lighter when you hover over them as it does not give sufficient colour contrast
- the colours on the homepage have divided opinion. Personally I really like it but a couple of people don't like the red on blue. Maybe we could have a chat about it on Friday morning?

**Comment by seenanair** [comment]
@stevieing 

> * would it be possible for the traction text to be white to give sufficient colour contrast
Yes 👍 
> * would it be possible for the link text to be lighter when you hover over them as it does not give sufficient colour contrast
I will do that.
> * the colours on the homepage have divided opinion. Personally I really like it but a couple of people don't like the red on blue. Maybe we could have a chat about it on Friday morning?
 May be white on those boxes - anyway we can discuss it on Friday
